bc_obps/Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN useradd -ms /bin/bash -d ${HOME} --uid ${USER_ID} -g root bc_obps
 
 COPY ./ ${HOME}/
 
+RUN ls
```

Comment:
Do we still need this line? I'm guessing it was added during debugging...

Reply:
💯%✅